### PR TITLE
Add proper styling to DlnaProfile and ScheduledTask pages

### PR DESCRIPTION
**Pull request by grafixeyehero**
Before
Screenshot_2019-06-01 Jellyfin(1)

After
Screenshot_2019-06-01 Jellyfin

### CODE REVIEW

File: src/components/dialogHelper/dialoghelper.css [hunk 1]
```diff
@@ -139,8 +139,8 @@
     }
 
     .dialog-small {
-        width: 60%;
-        height: 80%;
+        width: 100%;
```
Comment (DrPandemic) [inline src/components/dialogHelper/dialoghelper.css]: Something seems a little bit weird here, `.dialog-small` becomes bigger than `.dialog-medium`. Maybe you should use another class or rename this one?

**Comment by grafixeyehero** [comment]
> 1
> image
> 
> 2
> image
> 
which one did u prefer?

**Comment by DrPandemic** [comment]
This page is already weird with all the bullet points, but disregarding that, I think 1 is more readable.

**Comment by grafixeyehero** [comment]
@DrPandemic way u don't get the same rendering on firefox and chrome. b/c on firefox the emby-button inject button-inline class for this i will open a new PR that fix for both. And for the bullet points i will remove it only it appere on the title

**Comment by grafixeyehero** [comment]
> 
> 
> Except for the `button-link-inline` this looks good!

See PR  #356 Fix some style

**Pull request merged**